packages/web/src/components/search/DataSearch.js

```diff
@@ -428,7 +439,13 @@ class DataSearch extends Component {
 										})}
 										themePreset={themePreset}
 									/>
+									{
```

Comment:
Instead of ternary syntax here `condition ? component : null` we can use `&&` to avoid the `else` part of returning `null`. For example:
```js
{
    this.state.currentValue && this.props.showClear && <Component />
}
```